oidcgin/gin.go

```diff
@@ -20,13 +20,26 @@ func New[T any](claimsValidationFn options.ClaimsValidationFn[T], setters ...opt
 	return toGinHandler(oidcHandler.ParseToken, setters...)
 }
 
-func onError(c *gin.Context, errorHandler options.ErrorHandler, statusCode int, description options.ErrorDescription, err error) {
-	if errorHandler != nil {
-		errorHandler(description, err)
+func onError(c *gin.Context, errorHandler options.ErrorHandler, statusCode int, description options.ErrorDescription, err error) error {
+	if errorHandler == nil {
+		return c.AbortWithError(statusCode, err)
 	}
 
-	//nolint:errcheck // false positive
-	c.AbortWithError(statusCode, err)
+	error := options.OidcError{
```

Comment:
I'm not sure what I think about using `error` as the variable name. Could we use something like `oidcErr`?

Reply:
Well, `err` is the Golang idiom for an `*error` which this is not. I'm even ambivalent about calling the type `Error`, but `Context` is taken :cry:

Reply:
Fixed.